mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -2,13 +2,14 @@
 package google
 
 import (
-	"encoding/json"
+	"context"
 	"fmt"
 	"os"
+	"time"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google-beta/version"
```

Comment:
I think we want to remove this part and rely on it getting generated by `goimports`. It's intentionally omitted, and as-is it'll import the beta provider's version inside of the GA provider (which won't quite work)

We could also version-guard it, but since we're already running `goimports` it won't change anything.

Reply:
Ahhh thanks for pointing out goimports. I think I had something wrong with my workflow, after some fiddling I believe goimports is working correctly now and the generated code is behaving much better now.